storagenode/storagenodedb/storagenodesqldb.go

```diff
@@ -3,25 +3,25 @@
 
 package storagenodedb
 
-// versions represents the database that contains the database schema version history.
-type versionsDB struct {
-	location string
+type storageNodeSQLDB struct {
```

Comment:
Stutter, the full name of this type is `storagenodedb.storageNodeSQLDB`

Reply:
Fixed! Renamed to `migratableDB`.